src/clib/spio_io_summary.h

```diff
@@ -5,15 +5,15 @@
 #include "pio.h"
 #include "pio_internal.h"
 
-#define GPTL_TIMER_MAX_NAME 63
+#define MPI_TIMER_MAX_NAME 63
```

Comment:
Its the GPTL timers that have this restriction on the size. We want to be able to support GPTL timers in future.
So, can you change this constant to SPIO_TIMER_MAX_NAME and add a comment that GPTL timers assume this as the max name sz?

Reply:
Irrelevant, but the version of GPTL used in E3SM (in cime/src/share/timing/) has 

     #define MAX_CHARS 127

currently.

Reply:
OK, I have rebased this PR with suggested changes.